features/networking/sdn.feature

```diff
@@ -340,4 +340,34 @@ Feature: SDN related networking scenarios
       | curl | --connect-timeout | 5 | <%= cb.pod4_ip %>:8080 |
     Then the step should succeed
     And the output should contain "Hello OpenShift"
-
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-25787
+  @admin
+  Scenario: Don't write CNI configuration file until ovn-controller has done at least one iteration
+    Given the env is using "OVNKubernetes" networkType
+    And I select a random node's host
+    #Checking controller iteration 1
+    When I run command on the "<%= node.name %>" node's sdn pod:
+      | bash | -c | ovn-appctl -t ovn-controller connection-status |
+    Then the step should succeed
+    And the output should contain "connected"
+    #Checking controller iteration 2
+    When I run command on the "<%= node.name %>" node's sdn pod:
+      | bash | -c | ls -l /var/run/ovn/ |
+    Then the step should succeed
+    And evaluation of `@result[:response].match(/ovn-controller.\d*\.ctl/)[0]` is stored in the :controller_pid_file clipboard
+    #Checking controller iteration 3
+    When I run command on the "<%= node.name %>" node's sdn pod:
+      | bash | -c | ovn-appctl -t /var/run/ovn/<%= cb.controller_pid_file %> connection-status |
+    Then the step should succeed
+    And the output should contain "connected"
+    #Checking controller iteration 4
+    When I run command on the "<%= node.name %>" node's sdn pod:
+      | bash | -c | ovs-ofctl dump-flows br-int \| wc -l |
+    Then the step should succeed
+    And the expression should be true> @result[:response].match(/\d*/)[0].to_i > 0
+    #Checking final iteration post all above iterations passed. In this iteration we expect CNi file to be created
+    And I run commands on the host:
+      | ls /var/run/multus/cni/net.d/ovn-kubernetes.conf |
```

Comment:
just failed for me.
```
ls: cannot access '/var/run/multus/cni/net.d/ovn-kubernetes.conf': No such file or directory
```

```
sh-4.4# find  /var/run/multus/cni/net.d -ls
    23600      0 drwxr-xr-x   2  root     root           60 Mar 13 14:41 /var/run/multus/cni/net.d
   205684      4 -rw-------   1  root     root           94 Mar 13 14:41 /var/run/multus/cni/net.d/10-ovn-kubernetes.conf
```
`4.4.0-0.nightly-2020-03-13-073111`

Reply:
Thanks Ross for catching it. Yea it needs to be 10-ovn-kubernetes.conf which was right earlier before fix to this step. I am wondering why ```ls /var/run/multus/cni/net.d/ovn-kubernetes.conf``` is passing in automation despite it complains "ls: cannot access '/var/run/multus/cni/net.d/ovn-kubernetes.conf': No such file or directory". May be that's returning the exit code 0 based on . usuallu executing command in pod return exit status. Anyways i reverted back to my earlier commit which was specifically checking 10.ovn-kubernetest.conf for now it works okay now http://pastebin.test.redhat.com/844704